libs/components/lookup/src/lib/modules/lookup/lookup.component.ts

```diff
@@ -793,24 +793,24 @@ export class SkyLookupComponent
     // The input should NOT be focused if other elements (tokens, etc.)
     // are currently focused or being tabbed through.
 
-    observableFromEvent<MouseEvent>(documentObj, 'mousedown')
+    observableFromEvent<MouseEvent>(documentObj, 'click')
       .pipe(takeUntil(this.#idle))
-      .subscribe((event) => {
+      .subscribe(() => {
         hostElement = !this.inputBoxHostSvc
           ? this.#elementRef.nativeElement
           : this.lookupWrapperRef?.nativeElement;
-        this.isInputFocused = hostElement.contains(event.target);
+        this.isInputFocused = hostElement.contains(document.activeElement);
```

Comment:
We can now use the `activeElement` to be more precise because we no longer allow the blur to fire unless the click is outside of the dropdown.